src/DescopeKit.swift

```diff
@@ -54,6 +57,9 @@ public enum Descope {
     /// Authentication with TOTP
     public static var totp: DescopeTOTP { sdk.totp }
     
+    /// Authentication with passwords
```

Comment:
The indentation seems off here, I'm looking through the GitHub app so maybe a false positive?